Document administrators (#169)
servo/project: governance/README.md

People will be removed from this team after one year of no activity, and would be recognized as **Emeritus TSC Members**. A majority of **TSC Members** can also vote to remove a **TSC Member**.

## Administrators

> This is a smaller group of people group of people (maximum 5) that have administrator permissions in the Servo organization and repositories. Administrators would be also the [organization owners](https://github.com/orgs/servo/people?query=role%3Aowner).

Reviewer: ```suggestion
This is a smaller group of people group of people (maximum 5) that have administrator permissions in the Servo organization and repositories. Administrators are also assigned [ownership](https://github.com/orgs/servo/people?query=role%3Aowner) of the Servo GitHub organization.
```

> [Linux Foundation Europe](https://github.com/LF-Europe) would be always an extra administrator and owner while the project is hosted there.

Reviewer: ```suggestion
In addition, as long as [Linux Foundation Europe](https://github.com/LF-Europe) hosts the project, it is also included in the list of administrators as well.
```

### Requirements

> To be recognized with **Administrator** status, a community member must demonstrate to be trusted as a **TSC Member** of the Servo project for a significant amount of time.

Reviewer 2: So administrator is not necessarily TSC member, it is just trusted as a TSC member?

Author: Mmm, maybe the wording is wrong, I think the original idea was that administrators have to be TSC members. They have to be a TSC member for a significant amount of time which is trusted by the Servo community.

What if we reword it to:
```suggestion
To be recognized with **Administrator** status, a community member must be a **TSC Member** of the Servo project for a significant amount of time who has demonstrated to be trusted by the Servo community.
```

Reviewer 2: Maybe instead of "community member" we should just say "TSC Member"?

Disclaimer: I am also not native speaker.

> To be recognized with **Administrator** status, a community member must be a **TSC Member** of the Servo project for a significant amount of time who has demonstrated to be trusted by the Servo community.

Reviewer: ```suggestion
To be assigned **Administrator** status, a community member must have been a **TSC Member** for a significant amount of time and have the trust of the Servo community.
```

### Nomination

> If there's an available slot in the group a **TSC Member** nominates a person in the private *TSC* channel in Zulip explaining the reasons why that person should become **Administrator**. A majority of the **TSC Members** must approve the addition. If the vote succeeds and no objections are raised within one week, the person is added to the **Administrators** team.

Reviewer: ```suggestion
If there's an available administrator slot, a **TSC Member** can nominate a new administrator in the private *TSC* Zulip channel. The nomination should explain why that person should become **Administrator**. A majority of the **TSC Members** must approve the addition. If the vote succeeds and no objections are raised within one week, the person is added to the **Administrators** team.
```

### Removal

> People will be removed from this team after one year of no activity. A majority of **TSC Members** can also vote to remove an **Administrator**.

Reviewer: ```suggestion
Administrators can be removed in several ways:
 - They are automatically removed after one year of inactivity.
 - A majority of the **TSC** can vote to remove an **Administrator**.
 - Any **Administrator** can choose to step down by sending a request to the private *TSC* Zulip channel.
```
